apple/Filters/RNSVGFeGaussianBlur.mm

```diff
@@ -0,0 +1,129 @@
+#import "RNSVGFeGaussianBlur.h"
+
+#ifdef RCT_NEW_ARCH_ENABLED
+#import <React/RCTConversions.h>
+#import <React/RCTFabricComponentsPlugins.h>
+#import <react/renderer/components/rnsvg/ComponentDescriptors.h>
+#import <react/renderer/components/view/conversions.h>
+#import "RNSVGConvert.h"
+#import "RNSVGFabricConversions.h"
+#endif // RCT_NEW_ARCH_ENABLED
+
+@implementation RNSVGFeGaussianBlur
+
+#ifdef RCT_NEW_ARCH_ENABLED
+using namespace facebook::react;
+
+- (instancetype)initWithFrame:(CGRect)frame
+{
+  if (self = [super initWithFrame:frame]) {
+    static const auto defaultProps = std::make_shared<const RNSVGFeGaussianBlurProps>();
+    _props = defaultProps;
+  }
+  return self;
+}
+
+#pragma mark - RCTComponentViewProtocol
+
++ (ComponentDescriptorProvider)componentDescriptorProvider
+{
+  return concreteComponentDescriptorProvider<RNSVGFeGaussianBlurComponentDescriptor>();
+}
+
+- (void)updateProps:(Props::Shared const &)props oldProps:(Props::Shared const &)oldProps
+{
+  const auto &newProps = static_cast<const RNSVGFeGaussianBlurProps &>(*props);
+
+  self.in1 = RCTNSStringFromStringNilIfEmpty(newProps.in1);
+  self.stdDeviationX = [NSNumber numberWithFloat:newProps.stdDeviationX];
```

Comment:
Is having both coordinates for stdDeviation part of the standard?

Reply:
Yes, according to the W3 specs, there is a `stdDeviationX` as well as `stdDeviationY` https://www.w3.org/TR/SVG11/filters.html#InterfaceSVGFEGaussianBlurElement